File: JS/Fibonacci.js
function generateFibonacciSeries(n) {
    const series = [0, 1];

    for (let i = 2; i < n; i++) {
        const next = series[i - 1] + series[i - 2];
        series.push(next);
    }

    return series;
}

function printFibonacciSeries(series) {
    console.log("Fibonacci Series: " + series.join(", "));
}

const n = 10; // Change this to the number of terms you want
const fibonacciSeries = generateFibonacciSeries(n);
printFibonacciSeries(fibonacciSeries);

File: C/Fibonacci.c
#include <stdio.h>

int main() {
    int n, i;
    unsigned long long first = 0, second = 1, next;

    printf("Enter the number of terms: ");
    scanf("%d", &n);

    printf("Fibonacci Series: ");

    for (i = 1; i <= n; ++i) {
        if (i == 1) {
            printf("%llu, ", first);
            continue;
        }
        if (i == 2) {
            printf("%llu, ", second);
            continue;
        }

        next = first + second;
        first = second;
        second = next;

        printf("%llu, ", next);
    }

    printf("\n");

    return 0;
}
